pom.xml

```diff
@@ -70,6 +70,7 @@
   <modules>
     <module>sdk</module>
     <module>examples</module>
+    <module>contrib/hadoop</module>
```

Comment:
OK. Can you confirm that the contrib modules will be made available from Maven central?